Fixed wenz14jgr (#1562)
ESMValGroup/ESMValTool: doc/sphinx/source/recipes/recipe_wenzel14jgr.rst

Emergent constraints on carbon cycle feedbacks
==============================================

Overview
--------

> Figures from Wenzel et al. (2014) are reproduced with namelist_wenz14jgr.yml. Variables relevant for the carbon cycle - climate feedback such as near surface air temperature (tas), net biosphere productivity (nbp) and (fgco2) are analyzed for coupled (1pctCO2, here the carbon cycle is fully coupled to the climate response) and uncoupled (esmFixCLim1, here the carbon cycle is uncoupled to the climate response) simulations. The standard namelist includes a comparison of cumulated nbp from coupled and uncoupled simulations and includes a set of routines to diagnose the long-term carbon cycle - climate feedback parameter (GammaLT) from an ensemble of CMIP5 models. Also included in the namelist is a comparison of the interannual variability of nbp and fgco2 for historical simulations used to diagnose the observable sensitivity of CO2 to tropical temperature changes (GammaIAV). As a key figure of this namelistrecipe, the diagnosed values from the models GammaLT vs. GammaIAV are compared in a scatter plot constituting an emergent constraint.

Reviewer: ```suggestion
Figures from Wenzel et al. (2014) are reproduced with recipe_wenzel14jgr.yml. Variables relevant for the carbon cycle - climate feedback such as near surface air temperature (tas), net biosphere productivity (nbp) and carbon flux into the ocean (fgco2) are analyzed for coupled (1pctCO2, here the carbon cycle is fully coupled to the climate response) and uncoupled (esmFixCLim1, here the carbon cycle is uncoupled to the climate response) simulations. The standard namelist includes a comparison of cumulated nbp from coupled and uncoupled simulations and includes a set of routines to diagnose the long-term carbon cycle - climate feedback parameter (GammaLT) from an ensemble of CMIP5 models. Also included in the recipe is a comparison of the interannual variability of nbp and fgco2 for historical simulations used to diagnose the observable sensitivity of CO2 to tropical temperature changes (GammaIAV). As a key figure of this recipe, the diagnosed values from the models GammaLT vs. GammaIAV are compared in a scatter plot constituting an emergent constraint.
```
Fix namelist to recipe and explain fgco2 if you explain all the other variables

Available recipes and diagnostics
-----------------------------------

> Namelists are stored in recipes/

Reviewer: ```suggestion
Recipes are stored in recipes/
```

* recipe_wenzel14jgr.yml

Diagnostics are stored in diag_scripts/

* carbon_tsline.ncl: time line plots of annual means for spatial averages
* carbon_gammaHist.ncl: scatter plot of annual mean anomalies of two different variables; diagnosing and saving GammaIAV
* carbon_constraint.ncl: scatter plot of GammaLT vs. GammaIAV + line plot of probability density functions, diagnosing GammaLT

User settings
-------------

User setting files (cfg files) are stored in nml/cfg_carbon/

#. carbon_tsline 

   *Required Settings (scripts)*

   * ts_minlat: minimum latitude for area averaging
   * ts_maxlat: maximum latitude for area averaging
   * ts_minlon: minimum longitude for area averaging
   * ts_maxlon: maximum longitude for area averaging
   * ts_maxyear: last year (time range)
   * ts_minyear: first year (time range)
   * plot_units: units to appear on Figure
   * time_avg: currently, only yearly is available

>    * area_opper: type of area operation (sum)ridx_start: 0

Reviewer: ```suggestion
   * area_opper: type of area operation (sum)
```

   * styleset: Plot style

   *Optional settings (scripts)*

   * multi_model_mean: True for multi-model mean calculation

>    * volcanoes: True for marking years with lage volcanik erruptions

Reviewer: ```suggestion
   * volcanoes: True for marking years with lage volcanic eruptions
```

>    * align: True for aligning models to have the same start yeart (needed for idealized 2x CO2 Simulations)

Reviewer: ```suggestion
   * align: True for aligning models to have the same start year (needed for idealized 2x CO2 simulations)
```

   * ts_anomaly: calculates anomalies with respect to a defined time range average (anom)
   * ridx_start: if ts_anomaly is True, define start time index for reference period
   * ridx_end: if ts_anomaly is True, define end time index for reference period
   * ref_start: if ts_anomaly is True, define start year for reference period
   * ref_end: if ts_anomaly is True, define end year for reference period

   *Required settings (variables)*

   * reference_dataset: name of reference data set

#. carbon_gammaHist.ncl 

   *Required Settings (scripts)*
 
   * start_year: first year (time range)
   * end_year: last year (time range)
   * plot_units: units to appear on Figure
   * ec_anom: calculates anomalies with respect to the first 10-year average (anom) 

>    * scatter_log: set logaritmic axes in scatterplot.ncl

Reviewer: ```suggestion
   * scatter_log: set logarithmic axes in scatterplot.ncl
```

   * styleset: Plot style

   *Optional settings (scripts)*

>    * ec_volc : exklude 2 years after volcanic erruptions (True/False)

Reviewer: ```suggestion
   * ec_volc : exclude 2 years after volcanic erruptions (True/False)
```
